.github/workflows/test.yml

```diff
@@ -57,13 +59,34 @@ jobs:
       with:
         go-version: 1.14.x
     - uses: actions/checkout@v2
-    - run: |
-        go get github.com/x-motemen/gobump/cmd/gobump
-        go get github.com/mackerelio/golib/cmd/mackerel-github-release
-        make clean build rpm deb
-        mackerel-github-release && mackerel-github-release -staging
-      env:
-        GITHUB_TOKEN: ${{ secrets.MACKERELBOT_GITHUB_TOKEN }}
+    - run: make clean build rpm deb
+    - uses: actions/upload-artifact@v2
+      with:
+        name: linux-build-artifacts
+        path: |
+          ~/rpmbuild/RPMS/*/*.rpm
+          packaging/*.deb
```

Comment:
[releases](https://github.com/mackerelio/mackerel-agent-plugins/releases) is only `*.rpm`, `*.deb`